framework/src/main/java/org/checkerframework/common/basetype/messages.properties

```diff
@@ -39,6 +39,7 @@ contracts.precondition.expression.parameter.name=The precondition %s on the decl
 contracts.postcondition.expression.parameter.name=The postcondition %s on the declaration of method '%s' refers to formal parameter '%s' by name. Please use "#%s" instead of "%s".
 contracts.conditional.postcondition.expression.parameter.name=The conditional postcondition %s on the declaration of method '%s' refers to formal parameter '%s' by name. Please use "#%s" instead of "%s".
 
+super.invocation.invalid=Constructor of type %s cannot call super() of type %s.
```

Comment:
Couldn't the `super()` call have arguments?
The type you give as third argument is the constructor return type. Maybe instead use the full signature `constructorType`?

Reply:
Although people often write `()` just to mean "it's a method", I agree with making overloads explicit.  Thanks for the suggestion.